packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -88,6 +92,7 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
   const [position, setPosition] =
     useState<CSSProperties['position']>('relative');
   const router = useRouter();
+  const layout = useFeature(feature.feedLayout);
```

Comment:
Same here? Sorry for these, just trying to evaluate if that hook is necessary or not, if we're not using it.
Or is it because here we don't care on what feed we are in?

Reply:
Yep, this part is regardless of which feed you are in since this is the post page which is never in a feed.